src/firebase.ts

```diff
@@ -64,3 +75,13 @@ export function useEmulatedFirebaseApp(
 
   return app;
 }
+
+const loadCloudWorkstationCookies = async (url?: string) => {
+  if (url && url.includes('cloudworkstations.dev')) {
+    try {
+      const res = await fetchMaybeWithCredentials(url);
```

Comment:
Something I'm considering down the line is exponential backoff for the situation when the user hasn't started the emulator yet